tests/behat/theme_boost_union_block_region.feature

```diff
@@ -0,0 +1,553 @@
+@theme @theme_boost_union  @theme_boost_union_block_config @javascript
+Feature: Theme boost union block config
+  In order to add more functionality to pages
+  I need to add blocks to pages
+
+  Background:
+    Given the following "users" exist:
+      | username | firstname | lastname | email |
```

Comment:
Similar to https://github.com/moodle-an-hochschulen/moodle-theme_boost_union/blob/master/tests/behat/theme_boost_union_feelsettings_navigation.feature#L7-L18, please create only the users which you really need in this feature file and strive to vertically align the pipe symbols.